modules/java8/src/main/scala/io/circe/java8/time/TimeInstances.scala

```diff
@@ -36,6 +37,17 @@ trait TimeInstances {
 
   implicit final val encodeInstant: Encoder[Instant] = Encoder.instance(time => Json.fromString(time.toString))
 
+  implicit final val decodeZoneId: Decoder[ZoneId] =
+    Decoder[String].emap{
+      case s if ZoneId.getAvailableZoneIds.contains(s) =>
```

Comment:
I know this might be silly, but the timezone database is a big horrible global mutable thingy, and I'd rather have this instance follow the `try Right(...)` pattern of the others, so that it doesn't throw an exception if somehow the database changes between these two lines.